keps/sig-windows/2258-node-service-log-viewer/README.md

```diff
@@ -209,36 +213,42 @@ to appropriate resource type and associated endpoints, it will allow us to
 is setup in that manner. Access to the `node/logs` sub-resource needs to be
 explicitly granted as a user with access to `nodes` will not automatically have
-access to `node/logs`.
+access to `node/logs`. In the alpha phase the functionality will be behind
+`kubectl alpha node-logs` sub-command. The functionality will be moved to
+`kubectl logs` in the beta phase. However the examples will reference the final
+destination i.e. `kubectl logs`.
 
-The `logs` sub-command for node objects will follow a heuristics approach when
+The `logs --query` sub-command for node objects will follow a heuristics approach when
 asked to query for logs from a Windows or Linux service. If asked to get the
 logs from a service `foobar`, it will first assume `foobar` logs to the Linux
 journal / Windows eventing mechanisms (Application, System, and ETW). If unable
 to get logs from these, it will attempt to get logs from `/var/log/foobar.log`,
 `/var/log/foobar/foobar.log`, `/var/log/foobar*INFO` or
-`/var/log/foobar/foobar*INFO` in that order.
+`/var/log/foobar/foobar*INFO` in that order. Alternatively an explicit file
+location can be passed to the `--query` option.
 Here are some examples and explanation of the options that will be added.
 ```
 Examples:
   # Show kubelet logs from all masters
-  kubectl logs nodes --role master -s kubelet
+  kubectl logs nodes --role master -q kubelet
 
   # Show docker logs from Windows nodes
-  kubectl logs nodes -l kubernetes.io/os=windows -s docker
+  kubectl logs nodes -l kubernetes.io/os=windows -q docker
+
+  # Show foo.log from Windows nodes
+  kubectl logs nodes -l kubernetes.io/os=windows -q /foo/foo.log
 
 Options:
-      --case-sensitive=true: Filters are case sensitive by default. Pass --case-sensitive=false to do a case insensitive filter.
   -g, --grep='': Filter log entries by the provided regex pattern. Only applies to node journal logs.
   -o, --output='': Display journal logs in an alternate format (short, cat, json, short-unix). Only applies to node journal logs.
       --raw=false: Perform no transformation of the returned data.
       --role='': Set a label selector by node role.
   -l, --selector='': Selector (label query) to filter on.
-      --since='': Return logs after a specific ISO timestamp or relative date. Only applies to node journal or Get-WinEvent logs.
+      --since-time='': Return logs after a specific ISO timestamp or relative date. Only applies to node journal or Get-WinEvent logs.
       --tail=0: Return up to this many lines (not more than 100k) from the end of the log. Only applies to node journal or Get-WinEvent logs.
       --sort=timestamp: Interleave logs by sorting the output. Defaults on when viewing node journal logs.
-  -s, --service=[]: Return log entries from the specified service(s).
-      --until='': Return logs before a specific ISO timestamp or relative date.
+  -q, --query=[]: Return log entries from the specified service(s).
```

Comment:
since this is a list, you should clarify whether it is AND or OR logic - I think it is OR?  E.g. `-q foo -q bar` means all logs matching either foo OR bar

Reply:
OK, I will clarify that.